Use cases/UseCases.md

```diff
@@ -135,6 +138,22 @@ Results in a response with a cause of death registgration summary like this:
 }
 ```
 
+The cause of death registration is currently available as a legal document "Legeerklæring" PDF in Elektronisk dødsmelding Web at https://{environment}.helsepunkt.no/dodsmelding/mineregistreringer. This document is only available for the doctor who completed the registration.
+
+## Change time of death on existing death message
+A change in time of death on a Patient can be done by any authorized doctor. 
+It can be done following the same steps as for register death message only the Patient will during search Patient by id or by identifier be registered as dead. During create of Patient enter the new time of death. 
+
+## Abort death message
+If a person by mistake is reported as dead, it is important to abort the death message as soon as possible. Only abort a death message if the person is not dead. Right after the abort message is sent the doctor must contact Norsk helsenett operasjonssenter by phone +47 73 56 59 99. This will trigger a abort procedure at Norsk helsenett. It is important to establish dialog between Norsk helsenett/Norwegian tax office and the reporting doctor to prevent the consequences of reporting a person dead that in fact is not. Especially if this happens outside working hours.
```

Comment:
a abort procedure -> an abort procedure
consequences of reporting a person dead that in fact is not -> consequences of reporting an alive person as dead.